src/plugins/filters/components/FilterColumn.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
Let's name this FilterValue.vue, since this is going to be applicable to Plot views as well as table views.